projects/core/src/cart/services/cart-config.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { CartConfig } from '../config/cart-config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CartConfigService {
+  constructor(protected config: CartConfig) {}
+
+  isSelectiveCartEnabled(): boolean {
+    return this.config.cart.selectiveCart.enabled;
```

Comment:
Maybe, just for sake of reliability, we could return correct value even if we don't have default config provided?

Reply:
Fixed